### ISSUE TITLE
Data validation is too harsh

### ISSUE THREAD

**Issue by dimaursu** [Raven24/diaspora-federation#8]
https://github.com/Raven24/diaspora-federation/blob/master/lib/diaspora-federation/salmon/encrypted_slap.rb#L115 

This is kinda harsh for developers, requiring to have author_id of type string. We can convert that on our own, with a simple `.to_s` ; I think it's more important to check if all the content of `author_id` are digits.

**Comment by Raven24** [comment]
yeah, you're probably right... that's one of my java-isms when I don't remember ruby supports duck typing ;)
